src/main/java/yihuibot/Main.java

```diff
@@ -0,0 +1,66 @@
+package yihuibot;
+
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import yihuibot.component.MainWindow;
+import yihuibot.exception.taskformat.IncorrectTaskFormatException;
+
+/**
+ * Start the application.
+ *
+ * @author Toh Yi Hui A0259080A
+ */
+public class Main extends Application {
+    private static final String DEFAULT_FILE_PATH = "data/task.txt";
+    private YihuiBot yihuiBot;
+
+    /**
+     * Constructor for a new Main, with parameter to specify where to read tasks info from.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... etc. (not Return or Returning etc.)
```suggestion
     * Constructs a new Main, with parameter to specify where to read tasks info from.
```